cmd/lit-af/chancmds.go

```diff
@@ -148,6 +180,102 @@ func (lc *litAfClient) FundChannel(textArgs []string) error {
 	return nil
 }
 
+func (lc *litAfClient) DualFund(textArgs []string) error {
+	if len(textArgs) > 0 && textArgs[0] == "-h" {
+		fmt.Fprintf(color.Output, dualFundCommand.Format)
+		fmt.Fprintf(color.Output, dualFundCommand.Description)
+		return nil
+	} else {
+		return fmt.Errorf(dualFundCommand.Format)
+	}
+}
+
+// Mutually fund a channel
+func (lc *litAfClient) DualFundChannel(textArgs []string) error {
+	if len(textArgs) > 0 && textArgs[0] == "-h" {
```

Comment:
We added a couple of handlers to reduce these calls, might be nice to have it over here after looking into the imports issue trey mentioned :)